src/cli.rs

```diff
@@ -217,6 +217,15 @@ pub struct ImportArgs {
     /// however that the `ship` command will still self-update by default.
     #[arg(short, long)]
     pub keep_self: bool,
+
+    /// Also import all locally installed packages into the configuration. This
+    /// means packages installed via `cargo install --path <path>` will be present
+    /// in the configuration.
+    ///
+    /// Default: `false`, i.e. exclude all packages installed via `cargo install --path <path>`
+    /// from the list of packages to install or update in the resulting configuration file.
+    #[arg(short = 'l', long)]
+    pub with_local: bool,
```

Comment:
Just thought about it, but wouldn't it be more consistent with the above option if it was named `keep_local` parsed from `-l/--keep-local` instead? In this case, maybe change the shorthand version of `--keep-self` to `-s`?